### ISSUE TITLE
It is not possible to initialize multiple NGL widgets on same page

### ISSUE THREAD

**Issue by ozcan** [arose/ngl#537]
Hello, 
Thank you for the amazing project. Using NGL is very fun, with good examples and well-designed API.

However, I encountered one little issue, when I try to initialize multiple NGL instances, after some number, three.js throws an error about WebGL contexts.

Here is an example: https://codepen.io/ozcanesen/pen/JvBpVO
Here is a related StackOverflow discussion: https://stackoverflow.com/questions/21548247/clean-up-threejs-webgl-contexts

**Comment by arose** [comment]
Yes, that is expected. The Browsers only allow a certain number of WebGL contexts at a time. Best is to reuse a give NGL.Stage instance.

See also http://nglviewer.org/ngl/grid.html (look at the source) for an approach.
